src/termdict/termdict.rs

```diff
@@ -101,7 +102,7 @@ fn open_fst_index(source: ReadOnlySource) -> fst::Map {
 /// The term dictionary contains all of the terms in
 /// `tantivy index` in a sorted manner.
 ///
-/// The `Fst` crate is used to assoicated terms to their
+/// The `Fst` crate is used to assoicate terms to their
```

Comment:
hehe there is still the typo :)

Reply:
ha! that's what I get. ;p

Reply:
can you fix the typo:

`assoicate` => `associate`